app/services/CachingService.scala

```diff
@@ -141,12 +149,11 @@ trait CachingService extends SessionCache with AppName with ServicesConfig {
       }
     }
 
-
-  def getUpdateRelationshipCachedData(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[UpdateRelationshipCacheData]] =
+  def getCachedDataForEligibilityCheck(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[EligibilityCheckCacheData]] =
```

Comment:
It is tested within the TransferServiceTest. There are no actual tests for the CacheService. This may need to be done as a separate ticket after some discussion.

Reply:
I see, yeah that needs another ticket to look at the tests